crypto/keys/der/der_test.go

```diff
@@ -68,9 +68,9 @@ func TestFromProto(t *testing.T) {
 			t.Parallel()
 
 			signer, err := FromProto(test.keyProto)
-			if gotErr := err != nil; gotErr != test.wantErr {
+			if (err != nil) != test.wantErr {
 				t.Fatalf("FromProto(%#v) = (_, %q), want (_, nil)", test.keyProto, err)
```

Comment:
This error message looks wrong - it assumes that we wanted `err == nil`, but that's not the case if `test.wantErr == true`.

Reply:
Reverted this change.